SPECS/kubernetes/kubernetes.spec

```diff
@@ -95,7 +95,8 @@ Pause component for Microsoft Kubernetes %{version}.
 %build
 # set version information using version file
```

Comment:
This comment no longer seems accurate; please update (or let me know why it's still correct).

Reply:
Ohh, thank you for catching this - sure, this is incorrect;
Fixed